tox.ini

```diff
@@ -1,49 +1,30 @@
 # The test environment and commands
 [tox]
-envlist = deps, check, test
+envlist = check, test
 skipsdist = True
 
 [testenv:check]
 description = Runs all formatting tools then static analysis (quick)
-deps =
-    --no-deps
-    -r requirements-dev.txt
+deps = flake8
 commands =
     #shed  # combines autoflake, black, isort, and pyupgrade
     flake8
 
-[testenv:test]
+[testenv]
 description = Run the tests
 deps =
-    --no-deps
-    -r requirements.txt
+     pytest
+     pytest-cov
+     pytest-mock
+     pytest-xdist
 commands =
     pip install -e .
     python -m pytest --tb=native --cov=terminusdb_client terminusdb_client/tests/{posargs} --cov-report xml:cov.xml
 passenv = TERMINUSX_TOKEN
 
 
-[testenv:deps]
```

Comment:
all of this was incredibly evil.
The whole point of a build system is to have a single authority for how the build is done. Tox already uses the normal build (through pip install), and yet here we're messing with an entirely different set of requirement files. Pointless and confusing at best, widely divergent at worst.